SU2_CFD/include/solver_structure.hpp

```diff
@@ -947,7 +944,8 @@ class CSolver {
    * \param[in] config - Definition of the particular problem.
    * \param[in] val_marker - Surface marker where the boundary condition is applied.
    */
-  virtual void BC_Engine_Inflow(CGeometry *geometry, CSolver **solver_container, CNumerics *conv_numerics, CNumerics *visc_numerics, CConfig *config, unsigned short val_marker);
+  virtual void BC_Supersonic_Inlet(CGeometry *geometry, CSolver **solver_container,
```

Comment:
For the time being... no problem... let's try to do it different next time. Thanks!